src/js/apps/forms/form/form_app.js

```diff
@@ -101,4 +134,36 @@ export default App.extend({
       }
     });
   },
+  startUpdateApp() {
+    this.stopChildApp('history');
+
+    const updateApp = this.startChildApp('update', {
+      region: this.getRegion('form'),
+      form: this.action.getForm(),
+      response: this.formResponses.first(),
+      action: this.action,
+      state: this.getState(),
+    });
+
+    this.listenTo(updateApp.getState(), 'change:historyResponseId', (state, historyResponseId) => {
+      this.setState('historyResponseId', historyResponseId);
+    });
+  },
+  startHistoryApp() {
+    this.stopChildApp('update');
+
+    const historyApp = this.startChildApp('history', {
+      region: this.getRegion('form'),
+      action: this.action,
+      response: this.formResponses.first(),
+      form: this.form,
+    });
+
+    this.listenTo(historyApp, {
+      stop() {
+        this.formResponses = this.action.getFormResponses();
```

Comment:
I understand why this is happening, but.. I dunno.. don't love it.    I think either we should just `getFormResponses()` everywhere instead of storing `formResponses` on `this` _or_ we should listen to the action's `change:_form_response` and `this.formResponses = this.action.getFormResponses();`

Thoughts?

Reply:
Ah yes, I meant to write a note about this. I just thought I'd throw it in there to see if we wanted it. Listening to the change event seems to be the cleanest solution to me.